source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -198,6 +245,62 @@ GLSSharpNavierStokesSolver<dim>::generate_cut_cells_map()
     }
 }
 
+template <int dim>
+bool
+GLSSharpNavierStokesSolver<dim>::cell_cut_by_p_exception(
```

Comment:
I would change the title of that function. Maybe cell_cut_by_p_absolute_distance ?
Or something like this.
Exception is very vague